### ISSUE TITLE
regtest ww3_tp2.15: ww3_prep run multiple times unnecessarily

### ISSUE THREAD

**Issue by ukmo-ccbunney** [NOAA-EMC/WW3#484]
**Describe the bug**
In regtest ww3_tp2.15 (input_rho), each WW3 program is called via a separate invocation of `run_test`. This includes 3 calls to `ww3_prep` whilst manually moving the ww3_prep_*.inp files to the work directory, i.e. from matrix.base:
https://github.com/NOAA-EMC/WW3/blob/c79f485d990143cbb52c93956f1d1d19fdbf3a2c/regtests/bin/matrix.base#L824-L837

However, the `run_test` script automatically processes all files names `ww3_prnc_*` in the input directory, so this is not required. Furthermore, as a result of the 3 calls to `run_test` using the `-p ww3_prnc` flag, each of the 3 input files in `input_rho` (RHO, WND, TAU) are actually processed 3 times.

I believe the lines above can be replaced by a single line thus, which is much cleaner and avoids duplicate runs of ww3_prnc:
```
echo "$rtst -s MPI  -i input_rho -w work_PR3_UQ_RHO_MPI -f -p $mpi -n $np $ww3 ww3_tp2.15" >> matrix.body
```

A test of this produces the same results.

The same applies to the SHRD version of this test.

**To Reproduce**
Run the ww3_tp2.15/input_rho regression test, then run the above one line version (targeting a different work directory).
Results should be the same.

**Expected behavior**
Results of regression test are same.

**Comment by aliabdolali** [comment]
@ukmo-ccbunney that's a valid point, what about ww3_grid and ww3_strt for the same ww3_tp2.15?
I think the run_test is capable of handling multiple strt and grids and does not need to use _-r_ option.

**Comment by ukmo-ccbunney** [comment]
> @ukmo-ccbunney that's a valid point, what about ww3_grid and ww3_strt for the same ww3_tp2.15?
> I think the run_test is capable of handling multiple strt and grids and does not need to use _-r_ option.

That's right - the explicit call to ww3_grid and ww3_strt is not needed. The whole section (13 lines) can be replaced by a single call to run_test.

Also - I note that for the MPI version, the post-processor programs are not being run (as the -S switch is used and they think that the regtest has aready finished).

**Comment by aliabdolali** [comment]
@ukmo-ccbunney can we fix it in #483 as this PR touches ww3_tp2.15?

**Comment by ukmo-ccbunney** [comment]
@aliabdolali: OK - I will ask @ukmo-nievesvaliente to include it in her PR.

**Comment by ukmo-ccbunney** [comment]
Fixed in #483
